jekyll/_docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: classic-docs
+title: Introduction to Continuous Integration and CircleCI
+short-title: Introduction to CircleCI
+categories: [getting-started]
+description: "An introduction to Continuous Integration, Continuous Deployment, and how CircleCI can help."
+---
+
+Typically CircleCI is used to do [Continuous Integration][wiki-ci] and Continuous Deployment. 
+
+## Continuous Integration 
+
+The general idea is to test all changes that you are making to your code base. You can accomplish this with: 
+
+* [Unit Tests][wiki-unittest]
+* [Integration Tests][wiki-inttest]
+* [Functional Tests][wiki-functest] (also known as End to End tests) 
+
+[Software Testing][wiki-codetest] in general is a huge topic with tons of concepts. The way that you write tests varies depending on what language and framework you are using.
+
+## Continuous Deployment 
+
+If your tests pass, then you can deploy your code to development, staging, production, etc... The specific way that you do this depends on what type of infrastructure you are deploying too. Some examples:
+
+* [AWS CodeDeploy][doc-awscd]
+* [AWS EC2 Container Service (ECS)][doc-awsecs]
+* [AWS S3][doc-awss3]
+* [Google Container Engine (GKE)][doc-gke]
+* [Heroku][doc-heroku]
+* [Deploy Using SSH][doc-sshdeploy]
+
+## How does CircleCI help you 
+
+Now that we have a basic context of what CI and CD is, we can talk about how CircleCI fits into this picture. CircleCI integrates with your version control system (either GitHub or Bitbucket) and automatically runs a series of steps every time that we detect a change to your repository (when you push commits or a PR is opened). 
```

Comment:
Now that we have some context on CI and CD, we can discuss how CircleCI fits into this flow.

CircleCI integrates with your version control system (GitHub/Bitbucket) and automatically runs a series of steps every time a change is detected in the repository (i.e. when you push commits or open a PR).